Fix UPGRADING documentation to add tables migration

pacoorozco opened the pull request:
I was trying to upgrade to v4 and I realized, after some tries, that there are new migrations in the new version that should be added.

Gummibeer reviewed (changes requested):
Hey, thanks for that one! 🎉

Could you split it?
I would expect the composer line directly after the v3 to v4 headline and the two migration ones in a migration only section.
This would make it clearer what is what.

pacoorozco commented:
Hi @Gummibeer 

I've addressed your comments, let me say that I've discovered that, due to #901, one of the migrations should be removed after publishing them, the latest `*_create_activity_log_table`.

Gummibeer commented:
Thanks! 🥳

Yes, that one is still open and should be fixed (so far I know) in the package tools base package.
